Source/Applications/openXDA/openXDA/Controllers/Workbench/DataFileController.cs

```diff
@@ -325,6 +329,40 @@ IEnumerable<DataRow> QueryFileGroupIDs(IEnumerable<int> ids)
             }
         }
 
+        [Route("Download/{fileID:int}"), HttpGet]
+        public HttpResponseMessage DownloadFile(int fileID)
```

Comment:
This is necessary for the subsequent suggestions.

```suggestion
        public async Task<HttpResponseMessage> DownloadFile(int fileID)
```